packages/plugin-core-example/test/mocks/DummyLauncher.mock.ts

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
Same here. It does not make sense that each plugin would have these dummies. The core would need them as well for its own tests

Reply:
I agree partly, but the mocks are meant to simulate certain behaviour, we might have a dummylauncher in the core that does specific things. Things we don't want to happen in the tests for the plugin.

In addition, since the core package does not export the mocks we can not import them here.
If we really want this the @syntest/core package would have to include all mock.ts files which does not seem like the right solution here.

Reply:
Removed.